tests/acceptance/bootstrap/PublicWebDavContext.php

```diff
@@ -216,7 +216,6 @@ public function renameFileFromPublicShare(string $fileName, string $toFileName,
 	 * @return void
```

Comment:
Changes in this file are just linting fixes.

Reply:
should be fine